### PR TITLE
Port and warn for base.EMPTYSTRING

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
Following the comments [here](https://github.com/softdevteam/pygrate2/pull/18), I forward ported `base64.EMPTYSTRING"

### CODE REVIEW

File: Lib/warnings.py [hunk 1]
```diff
@@ -513,78 +522,6 @@ def warn_explicit(message, category, filename, lineno,
     msg = WarningMessage(message, category, filename, lineno, source)
     _showwarnmsg(msg)
 
-def warn_explicit_with_fix(message, fix, category, filename, lineno,
```
Comment (ltratt) [inline Lib/warnings.py:516]: Why is this function deleted in this PR? Is it just a hangover from something previous?

Comment (nanjekyejoannah) [reply Lib/warnings.py:516]: Code was removed in error, reverted in [2798377](https://github.com/softdevteam/pygrate3/pull/11/commits/27983773d238f488888fe55989c2bbe146c77960)

**Comment by nanjekyejoannah** [comment]
bors try

**Comment by nanjekyejoannah** [comment]
This was squashed in error, sorry, the branch names confused me.

**Comment by ltratt** [comment]
Please force push an update back to the previous commit (I think  2798377 ?).

**Pull request closed without merging**